packages/api/src/services/received_emails.ts

```diff
@@ -0,0 +1,22 @@
+import { ReceivedEmail } from '../entity/received_email'
+import { getRepository } from '../entity/utils'
+
+export const saveReceivedEmail = async (
+  from: string,
+  to: string,
+  subject: string,
+  text: string,
+  html: string,
+  userId: string,
+  type: 'article' | 'non-article' = 'non-article'
```

Comment:
Does non-article include PDF attachment messages? I don't think we have to handle those specially, just wondering out loud.

Reply:
Yeah, it should include PDF attachment